### ISSUE TITLE
The list of cantons is wrong.

### ISSUE THREAD

**Issue by michelrossier** [NetConsultAG/netc_memberlist#12]
Please use the attached list for the dropdown values.

**Cantons old sgrm.ch webseite | Must be on the new website dev.sgrm.netconsult.ch | French translation**

AG = Aargau | Argovie
AI = Appenzell Innerrhoden | Appenzell Rh.-Int.
AR = Appenzell Ausserrhoden | Appenzell Rh.-Ext.
BE = Bern | Berne
BL = Basel-Landschaft | Bâle-Campagne
BS = Basel-Stadt | Bâle-Ville
FR = Freiburg | Fribourg
GE = Genf | Genève
GL = Glarus | Glaris
GR = Graubünden | Grisons
JU = Jura | Jura
LU = Luzern | Lucerne
NE = Neuenburg | Neuchâtel
NW = Nidwalden | Nidwald
OW = Obwalden | Obwald
SG = Sankt Gallen | Saint-Gall
SH = Schaffhausen | Schaffhouse
SO = Solothurn | Soleure
SZ = Schwyz | Schwytz
TG = Thurgau | Thurgovie
TI = Tessin | Tessin
UR = Uri | Uri
VD = Waadt | Vaud
VS = Wallis | Valais
ZG = Zug | Zoug
ZH = Zürich | Zurich

**Comment by nitsan-technologies** [comment]
Hi @michelrossier 

As per our last confirmation, we have used static_info_tables data for following field:
1. country = country
2. canton = territorry

Question: Do you need to change it from static_info_tables to new custom table - where you will manage it from backend?
OR You just want to change it from relation to static_info_tables to static values given by you for canton field only?

Looking forward to your reply here. Thank you!

**Comment by michelrossier** [comment]
Hi @nitsan-technologies 

Thanks for the feedback.

The "territory" are continents, but what we need are cantons (like provicnes of Switzerland).

What we need is basically the **static_country_zones** of static_info_tables, I just learend that this contains the given names in my initial ticket.

**Comment by nitsan-technologies** [comment]
Hi @michelrossier 

We have done this. Screenshot: http://prntscr.com/ceo1lj , FE: http://prntscr.com/ceo1cd

NOTE:- The canton list (static_country_zones) is not multi-lingual provided by static_info_tables, but the zone names are in their local languages.

Thank you!

**Comment by michelrossier** [comment]
Thanks. I see that it is working in your installation, but after I installed it it still does not reflect in my installation:

http://sgrm.netconsult.ch/de/allgemein/mitgliederbereich/meine-mitgliedschaft/ (see field "Kanton") it still shows the continents. 

Do I have to do anything else but installing your update from GitHub? I did compare and apply DB specifications and emptied the db and general cache. Thanks for your help.
